src/windows/track_place.c

```diff
@@ -445,6 +441,7 @@ static void window_track_place_attempt_placement(rct_track_td6 *td6, int x, int
  */
 static void window_track_place_paint(rct_window *w, rct_drawpixelinfo *dpi)
 {
+	set_format_arg(0, char *, _trackDesign->name);
```

Comment:
I don't think this is a very clever way of doing this. I much prefer the older way of assigning it to 3155.